tests/components/local_todo/test_init.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for init platform of local_todo."""
+
+from unittest.mock import patch
+
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_remove_config_entry(
```

Comment:
I'd also add test that setup and unloads the config entry and assert that entity state exist after setup and is unavailable after unload.